packages/aws-cdk/lib/init-templates/v1/app/csharp/add-project.hook.ts

```diff
@@ -3,10 +3,13 @@ import * as path from 'path';
 import { InvokeHook } from '../../../../init';
 
 export const invoke: InvokeHook = async (targetDirectory: string) => {
+  const escapeSpaces = (value: string) => {
+    return value.replace(' ', '\\ ');
```

Comment:
Does this also work on Windows? I'm pretty sure that if you put a `\` in a command line on Windows, it is just interpreted as a literal `\`, not as escaping the next character.